internal/artifacts/module_arch.go

```diff
@@ -142,12 +142,15 @@ func packModule(ep dir.IModule, deploymentDesc bool, module *mta.Module, moduleN
 			moduleName)
 	}
 
-	entry, err := os.Stat(buildResults)
-	if err != nil {
-		return errors.Wrapf(err, `packing the "%v" module failed; the "%v" build results path does not exist`,
-			moduleName, buildResults)
+	definedArchive := false
+	if buildResults != "" {
+		definedArchive, err = isArchive(buildResults)
+		if err != nil {
+			return errors.Wrapf(err, `packing of the "%v" module failed because could not find the "%s" build results`, moduleName, buildResults)
```

Comment:
Could you use this message? It's more similar to the previous one
`packing the "%v" module failed; could not read the "%s" build results path`